Resources/Prototypes/Tiles/floors.yml

```
@@ -1932,6 +1932,27 @@
   deconstructTools: [ Prying ]
   itemDrop: FloorTileItemAstroSnow
 
+#SS220-beachCraftng-Begin
+- type: tile
+  id: FloorAstroSnad
```

```suggestion
  id: FloorAstroSand
```
и исправить везде, где есть референс